app/models/pet.rb

```diff
@@ -1,7 +1,13 @@
 class Pet < ApplicationRecord
   belongs_to :shelter
-  validates_presence_of :image
-  validates_presence_of :name
-  validates_presence_of :approximate_age
-  validates_presence_of :sex
+  validates_presence_of :image, :name, :approximate_age, :sex
+
+  def adoption_status_text
+    if adoption_status == "adoptable"
+      "Adoption Pending "
+    else
+      "Adoptable"
+    end
```

Comment:
I was able to get this aspect of the new adoption update functionality working in the model!